kerascv/layers/losses/hard_neg_miner.py

```diff
@@ -0,0 +1,78 @@
+import tensorflow as tf
+
+
+class HardNegativeMiner(tf.keras.layers.Layer):
+    """Hard Negative Mining to keep balanced positive and negative samples."""
```

Comment:
Please add a more detailed docstring, including information about the constructor arguments, call inputs, and call return value

Reply:
Done.